### PR TITLE
Outline permission docs structure

### PR DESCRIPTION
**Pull request by Joonpark13**
## Hey, I just made a Pull Request!

This PR only scaffolds the file structure (and thereby the sidebar structure) of the permission related documentation.

#### :heavy_check_mark: Checklist

- [x] Added or updated documentation
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/permission/writing-a-policy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+id: writing-a-policy
+title: Writing a permission policy
+description: How to write your own permission policy as a Backstage integrator
```
Comment (mtlewis) [inline docs/permission/writing-a-policy.md:4]: nit: do we mean the same thing by "adopter" in the  getting started description and "Backstage integrator" here? If so, perhaps we should pick one and use it for both.

**Review by mtlewis — APPROVED** [review]
Love this! One small nit, no need to block merging.

File: docs/permission/writing-a-policy.md (hunk 1 above)
Comment (joeporpeglia) [reply docs/permission/writing-a-policy.md:4]: Is it necessary to specify the persona here? Could just leave it at "How to write your own permission policy"

Comment (Joonpark13) [reply docs/permission/writing-a-policy.md:4]: Good catch - sticking with integrator for now, but we can find all + replace later if we decide differently.

**Review by vinzscam — APPROVED** [review]

**Pull request merged**